templates/security/certifications.html

```diff
@@ -35,33 +35,43 @@ <h2 class="p-heading--3">Deploy regulated and high security workloads</h2>
+     <p>The default configuration of Ubuntu balances usability and security. However, systems carrying dedicated workloads can be further hardened to reduce their attack surface. Canonical works with DISA to ensure STIG guides are available for Ubuntu, and we provide OpenSCAP tooling and automation for the Industry accepted CIS benchmark. Available with <a href="/advantage">Ubuntu Advantage</a> and <a href="/public-cloud">Ubuntu Pro</a>.</p>
      <a class="p-button--neutral" href="#compliance-profiles">See our compliance profiles</a>
    </div>
    <div class="col-4 p-divider__block">
-     <h3>Transparency</h3>
-     <p>Canonical is transparent on the Security Features we enable on Ubuntu. We have a <a href="/security/notices">public vulnerability disclosure policy</a> and known vulnerabilities are disclosed in <a href="/security/oval">Machine readable OVAL</a> CVE output to be used by OpenSCAP and other 3rd party tools.</p>
+     <h3>Know your defenses</h3>
+     <p>Each Ubuntu release enables state of the art protection against vulnerability exploitation and malware and we <a class="p-link--external" href="https://wiki.ubuntu.com/Security/Features">publicly detail our choices</a>. Canonical has a <a href="/security/notices">public vulnerability disclosure policy</a> and vulnerabilities are not only fixed with automated security updates and <a href="/security/livepatch">livepatches</a> but also publicly disclosed with our <a href="/security/notices">security notices</a>. We further provide <a href="/security/oval">machine readable OVAL</a> CVE output to be used by OpenSCAP and other 3rd party vulnerability management tools.</p>
      <a class="p-button--neutral p-link--external" href="https://wiki.ubuntu.com/Security/Features">See our security features</a>
    </div>
    <div class="col-4 p-divider__block">
-     <h3>Security certification and 3rd party attestation</h3>
+     <h3>Access certifications for high security environments</h3>
      <p>Access to certification artifacts as well as the necessary tooling for regulated and high security environments. <a href="/advantage">Ubuntu Advantage</a> and <a href="/public-cloud">Ubuntu Pro</a> provide access to FIPS 140-2 certified cryptographic packages, allowing you to deploy workloads that need to operate under compliance regimes like FedRAMP, HIPAA, and PCI-DSS. Additionally, Ubuntu versions have been certified under Common Criteria, providing 3rd party attestation of the security mechanisms in the operating system.</p>
      <a class="p-button--neutral" href="#security-certifications">See our certifications</a>
    </div>
  </div>
 </section>
 
+<section class="p-strip">
+  <div class="row">
+    <div class="col-6">
```

Comment:
Is there a reason we only use half of the space?

Reply:
This should probs have a design needed label

Reply:
> Is there a reason we only use half of the space?

There should be a video in the other half I assume.

Reply:
Yes I was in discussion about what the PM wanted, added now.

Reply:
Yes I was in discussion about what the PM wanted, added now.